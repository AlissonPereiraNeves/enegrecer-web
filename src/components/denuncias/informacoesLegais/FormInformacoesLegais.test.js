import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import InformacoesLegais from './FormInformacoesLegais';

describe('DenuncianteForm', () => {
  it('deve seguir a estrutura definida ', () => {
    const informacoesLegais = shallow(<InformacoesLegais />);
    const informacoesLegaisJson = toJson(informacoesLegais);
    expect(informacoesLegaisJson).toMatchSnapshot();
  });
});
