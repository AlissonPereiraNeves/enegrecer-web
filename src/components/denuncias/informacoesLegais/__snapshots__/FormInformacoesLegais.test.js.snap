// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DenuncianteForm deve seguir a estrutura definida  1`] = `
<div>
  <br />
  <p>
    Informaçoes Legais
  </p>
  <br />
  <h4>
     Você conseguiu realizar um boletim de ocorrência? Ou tem informações caso você não seja a vítima? 
  </h4>
  <p>
    Essas informações são importantes para conseguirmos identificar a quantidade de vítimas que conseguem reportar esse crime. 
  </p>
  <div
    className="row"
  >
    <campoTexto
      divClasse="input-field col s6"
      id="numero-boletim"
      label="Número do Boletim de ocorrência"
      maxLen={40}
      type="text"
    />
    <campoTexto
      divClasse="input-field col s6"
      id="categoria-crime-boletim"
      label="Categoria do crime no BO"
      maxLen={40}
      type="text"
    />
  </div>
  <div
    className="row"
  >
    <campoTexto
      divClasse="input-field col s6"
      id="numero-processo"
      label="Número do Processo"
      maxLen={40}
      type="text"
    />
    <campoTexto
      divClasse="input-field col s6"
      id="orgao"
      label="Órgão"
      maxLen={40}
      type="text"
    />
  </div>
</div>
`;
